src/applications/claims-status/tests/e2e/fixtures/mocks/mock-api/claims/claims-list.json

```diff
@@ -88,6 +88,72 @@
         "updatedAt": "2023-03-30T18:19:29.857Z"
       }
     },
+    {
+      "id": "closed-with-documents-needed",
```

Comment:
Can we change this to `closed-w-documents-needed`? I realize I'm nitpicking but would be nice to maintain the consistency